src/OpenDiffix.Core.Tests/Aggregator.Tests.fs

```diff
@@ -19,16 +18,23 @@ let buildAidInstancesSequence numAids (random: System.Random) =
 let buildIntegerSequence (random: System.Random) =
   Seq.initInfinite (fun _ -> randomNullableInteger random)
 
-let makeArgs hasValueArg random numAids length =
-  let argsSeq =
-    if hasValueArg then
-      (buildAidInstancesSequence numAids random, buildIntegerSequence random)
-      ||> Seq.map2 (fun aidInstances argValue -> [ aidInstances; argValue ])
-    else
-      buildAidInstancesSequence numAids random
-      |> Seq.map (fun aidInstances -> [ aidInstances ])
-
-  argsSeq |> Seq.truncate length |> Seq.toList
+let makeAnonArgs hasValueArg random numAids length =
```

Comment:
> this test has a bunch of complex functions built on top of more functions

Yeah... way too complex. It basically generates a list of random arguments for the aggregates. Each transition can have multiple parameters such as `diffix_count(my_column, aid)` which are modelled as lists. Therefore the result is a list of lists. I will try to simplify if I can, otherwise this is a nice side task.

Reply:
The idea of these tests is to generate 2 sets of random arguments and verify if

```
agg2(args2) -> merge_to -> agg1(args1) == agg(args1 ++ args2) 
```

Reply:
thanks I think this is clearer now!